### ISSUE TITLE
[bug] Move cursor left after typing '+' sign inserts project name

### ISSUE THREAD

**Issue by metbril** [mjdescy/TodoTxtMac#126]
In the 'new task' box, if you enter a `+` sign and then move cursor left a few times, the first project name is inserted multiple times.

**Comment by mn113** [comment]
In fact if your cursor is anywhere within a tag and you start pressing left or right arrows, it tries to autocomplete and ends up inserting extra substrings from the tag.

Version 2.4.0 (2032)

**Comment by excetara2** [comment]
Yes I have this same problem for contexts or tags. I usually use alt+f and alt+b to navigate over words. If you press alt+b also it will autocomplete when it is within a tag.

**Comment by Gnossos** [comment]
Something like this also seems to happen when you edit a preexisting project string. E.g., if you have a project named "+ibmwebsite" and decide you want to change it to "+IBMwebsite," when you edit the task, backspace over the initial "ibm," and enter the new initial "I", the entire earlier project name will be inserted and you'll get "+ibmwebsiteBMwebsite" instead of what you want.
